uaclient/apt.py

```diff
@@ -377,6 +379,7 @@ def run_apt_update_command(
             cmd=["apt-get", "update"], override_env_vars=override_env_vars
         )
     except exceptions.APTProcessConflictError:
+        LOG.warning("Error running apt update")
```

Comment:
Should we capture the exception and log it here too ?

Reply:
Good catch! Will update this one